src/components/AlgoOrdersTable/AlgoOrdersTable.columns.js

```diff
@@ -40,6 +40,7 @@ export default (authToken, cancelOrder, gaCancelOrder) => [{
     <div className='icons-cell'>
       <i
         role='button'
+        aria-label='Remove order'
```

Comment:
keep bitfinex wordings i.e. Cancel order

```suggestion
        aria-label='Cancel order'
```